server/activityItem/queries.ts

```diff
@@ -118,9 +117,9 @@ export const createCommunityCreatedActivityItem = async (
 export const createCommunityUpdatedActivityItem = async (
 	actorId: null | string,
 	communityId: string,
-	oldCommunity: types.Community,
+	oldCommunity: Community,
```

Comment:
I felt comfortable changing these because only full models are passed to these functions, never the serialized one.
Also, the serialized one threw errors, because the `getDiffsForPayload` function generates a `Diff`ed type between the result of `Community.findOne` and `types.Community` (different types obv), which cannot get passed to `createActivityItem`.